content/docs/manage/integrations.md

```diff
@@ -2,43 +2,38 @@
 title: Manage integrations
 enableTableOfContents: true
 isDraft: false
-updatedOn: '2023-10-07T10:43:33.418Z'
+updatedOn: '2023-11-03T11:47:31.853Z'
 ---
 
-Neon supports integrations with partners such as Vercel. You can view and revoke integrations for a Neon project from the **Integrations** page in the Neon Console.
+The **Integrations** page in the Neon Console provides an interface for managing third-party integrations with your Neon project. Through Neon's supported integrations, you can optimize and extend Neon's functionality.
```

Comment:
```suggestion
The **Integrations** page in the Neon Console provides a hub for managing third-party integrations with your Neon project. Use these integrations to optimize and extend Neon's functionality and streamline your workflow.
```

Reply:
Added a little bit about "streamline your workflow" to capture that these integrations may have larger overall goals. There may be better wording, depending on the range of upcoming integrations.